includes/admin.inc.php

```diff
@@ -994,6 +994,64 @@
 		die();
 	}
 
+	if (isset($_GET['action']) and $_GET['action'] == 'list_uploads') {
+		$uploaded_images_path = 'images/uploaded/';
+		$images = array();
+		$browse_images = (isset($_GET['browse_images']) && $_GET['browse_images'] > 0) ? intval($_GET['browse_images']) : 1;
+		$handle = opendir($uploaded_images_path);
+		while ($file = readdir($handle)) {
+			if (preg_match('/\.jpg$/i', $file) || preg_match('/\.png$/i', $file) || preg_match('/\.gif$/i', $file)) {
+				$images[] = $file;
+			}
+		}
+		closedir($handle);
+		if ($images) {
+			rsort($images);
+			$page_browse['total_items'] = count($images);
+			$page_browse['items_per_page'] = $settings['uploads_per_page'];
```

Comment:
This is because you haven't the matching entry it the settings table. Should I add a line with a check for the existence of the setting and a backup for the case of a failing check?

~~~
$page_browse['items_per_page'] = (isset($settings['uploads_per_page'])) ? $settings['uploads_per_page'] : 20;
~~~

Reply:
>  Should I add a line with a check for the existence of the setting and a backup for the case of a failing check?

No, my fault, I'm sorry for this mistake. I only update the PHP files and not the SQL statement. I will add the SQL setting and re-test the scenario. Please ignore my comment. ;-)